index.md

```diff
@@ -238,3 +238,67 @@ Our typical usages of AWS include:
 1. S3 for hosting uploaded files
 1. ElastiCache instances for hosting in-memory data stores
 1. OpsWorks for provisioning each if the size or complexity is large
+
+# Documentation
+
+Traditionally documentation has been a second-class citizen in most Rails project. This holds true just as much outside The Nerdery as internally. As innovators and leaders in our industry we should hold ourselves to a higher standard and therefore provide ample documentation on the products we produce. A great article on improving Rails based documentation published by Aaron Sumner can be found [here](http://everydayrails.com/2016/04/18/rails-documentation-practices.html) and many of the practices described below are pulled from his philosophy.
+
+## Start with the README
+Your application’s README is still its gateway, the first thing new developers will likely read when they access its source code. If your app’s README consists of the default given to every new Rails app, it’s time to change it.
+
+Your README should include:
+
+* A brief description of what the application does.
+* How to install dependencies, especially those that are more complex than a simple `bundle install`. For example, if I need to to install Redis or ImageMagick to work on the app, or configure environment variables to talk to external services, let me know up front.
+* How to run the application’s test suite, to verify that setup was successful. Again, note special external dependencies, like PhantomJS.
+* How to generate the docs, to get to a more detailed description of the application’s code.
```

Comment:
Do we have links to good example README's, maybe OSS ones on GitHub for projects we admire?

Should we define a standard layout? Or is that too project-dependent to be generically useful? e.g. in my boilerplate readmes I usually set them up like so:
- Project title + description below
- Requirements
- Setup
- Common tasks
- Notes, reference files, links to other documentation